test/realm_test.dart

```diff
@@ -295,7 +308,7 @@ Future<void> main([List<String>? args]) async {
 
       expect(() {
         realm.write(() {
-          realm.add(Car()..make = "Tesla");
+          realm.add(Car(make: "Tesla"));
```

Comment:
I suggest we handle this better. A single field and a primary key shouldn't require a named field. This will get very frustrating when used.

Reply:
It is only because the "user" gave a default value for a primary key. Otherwise it would have been a mandatory unnamed argument. Again, I didn't rewrite all the tests. Also, I guess it is a corner-case we need to handle.

Reply:
Btw, name is not actually the primary key. `_Team` has an implicit primary key.

Reply:
Fixed